frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'global/state';
+
+import { OpenEdXInstanceDeploymentStatusStatusEnum as DeploymentStatus } from 'ocim-client';
+
+import { Col, Row } from 'react-bootstrap';
+
+import { getDeploymentsAlerts } from 'console/actions';
+import { DeploymentAlertModel } from 'console/models';
+import { WrappedMessage } from 'utils/intl';
+
+import messages from './displayMessages';
+import { ConsolePage } from '../ConsolePage';
+
+import './styles.scss';
+
+interface ActionProps {
+  getAlerts: Function;
+}
+interface Props extends ActionProps {
+  alerts: Array<DeploymentAlertModel>;
+}
+
+interface AlertProps {
+  alert: DeploymentAlertModel;
+}
+
+const iconClassMap = {
+  [DeploymentStatus.Healthy]: 'fa-check-circle',
+  [DeploymentStatus.ChangesPending]: 'fa-check-circle',
+  [DeploymentStatus.Provisioning]: 'fa-info-circle',
+  [DeploymentStatus.Preparing]: 'fa-info-circle',
+  [DeploymentStatus.Unhealthy]: 'fa-times-circle',
+  [DeploymentStatus.Offline]: 'fa-stop-circle'
+};
+
+const statusClassMap = {
+  [DeploymentStatus.Healthy]: 'healthy',
+  [DeploymentStatus.ChangesPending]: 'changes_pending',
+  [DeploymentStatus.Provisioning]: 'provisioning',
+  [DeploymentStatus.Preparing]: 'preparing',
+  [DeploymentStatus.Unhealthy]: 'unhealthy',
+  [DeploymentStatus.Offline]: 'offline'
+};
+
+const Alert: React.FC<AlertProps> = (props: AlertProps) => {
+  const [detailsCollapsed, setDetailsCollapsed] = React.useState(true);
+
+  const deploymentStatus = props.alert.status;
+
+  const caretClass = detailsCollapsed ? 'fa-caret-down' : 'fa-caret-up';
+  const collapsedClass = detailsCollapsed ? 'collapsed' : '';
+  const iconClass = iconClassMap[deploymentStatus];
+  const statusClass = statusClassMap[deploymentStatus];
+
+  const month = props.alert.date.toLocaleString('default', { month: 'long' });
+  const day = props.alert.date.getDate();
+  const year = props.alert.date.getFullYear();
+
+  const details = props.alert.deployedChanges ? (
+    JSON.stringify(props.alert.deployedChanges)
```

Comment:
Hey @giovannicimolin, do you have ideas about this?

PR is near to acceptable state, so if you have any suggestions, I can implement this.

Reply:
@Agrendalath @0x29a Sorry for missing this ping. 
My idea was to have a parsed text here with the values changed (nothing fancy, just not a RAW json file).
It can be a simple list with items like this:
- Changed `variable_name` to `#ffaa00`
- Changed `variable_name` to `#00ee33`

Do you know have an estimate of how hard it is to implement that? Do you have enough time allocated for this sprint for this? 
If not, then expand the budget a bit and move it to blocker, and finish it early next sprint.

_Note:_ ignore the color preview from github.

Reply:
@giovannicimolin 
Thanks!

Well, it appears much easier than I thought :+1:

I think I can finish this before end of day today.